### DISCUSSION TITLE
Custom admonition icons in mkdocs-material

### DISCUSSION THREAD

**Discussion by jasper-zanjani** [facelessuser/pymdown-extensions#1383] [Q&A] [answered]
I've recently learned how to make custom admonitions in [mkdocs-material](https://github.com/squidfunk/mkdocs-material), which uses pymdownx for parsing admonition syntax according to its maintainer . I am pretty satisfied with the results so far, but I was wondering if there is a way to specify a blank admonition title without defining whitespace. I've tried defining a blank `content` attribute for `.md-typeset .rs > summary`, to no avail.

![admonitions](https://user-images.githubusercontent.com/8889926/125080185-82d86980-e092-11eb-84a2-564d6bf68c59.png)

I'm currently only able to create this effect by defining a title with whitespace in the markdown:
```md
!!! rs "&nbsp;"
    <!-- snip -->
!!! cs "&nbsp;"
    <!-- snip -->
!!! py "&nbsp;"
    <!-- snip -->
```

**Comment by facelessuser** [comment, marked as answer]
There may be a little confusion here. Pymdown Extensions offers a plugin called Details that works like Admonitions (except that Details are collapsible).  The Details extension was inspired by Admonitions, but we do not distribute the Admonitions extension in our project. Also, styling is provided by themes like MkDocs Material or the user themselves. We do not offer any auto-styling.

So, if we are talking about Details, you can define an empty title:

```
???+ note ""
    Some Text
```

<img width="712" alt="Screen Shot 2021-07-09 at 7 10 33 AM" src="https://user-images.githubusercontent.com/1055125/125082727-ccba5300-e084-11eb-8dbf-3884e5b7353c.png">

As far as Admonitions, you can ask this over on the Python Markdown repo. It seems that currently if you specify an empty title (no whitespace), the title element is not inserted in the final HTML. If this is considered a bug by the maintainer, I'd be happy to create a fix for it as I am a member of that organization, but if the maintainer has a reason for this behavior, using a `&nbsp;` may be the appropriate workaround.
